app/scripts/views/recordtype/schema/edit-controller.js

```diff
@@ -105,8 +105,9 @@
                     disable_array_add: false,
                     theme: 'bootstrap3',
                     show_errors: 'change',
-                    no_additional_properties: true,
-                    startval: initialData
+                    startval: initialData,
+                    keep_oneof_values: false,
+                    prompt_before_delete: false,
```

Comment:
The default setting for the new version of JSONEditor is to prompt the user to confirm that they want to delete an element when they remove it from the form. This is in general nice behavior, but our form already doesn't persist changes unless the user hits "save," so it's actually more confusing to be prompted with a dialog box that doesn't actually control the deletion of the element.